shared/api/response.go

```diff
@@ -65,11 +65,7 @@ func (r *Response) MetadataAsStringSlice() ([]string, error) {
 
 // MetadataAsStruct parses the Response metadata into a provided struct
 func (r *Response) MetadataAsStruct(target interface{}) error {
-	if err := json.Unmarshal(r.Metadata, &target); err != nil {
-		return err
-	}
-
-	return nil
+	return json.Unmarshal(r.Metadata, &target)
```

Comment:
Uh, nice that they enforce this.

Reply:
Yeah, I just wonder how many more they have :)